taf/api/roles.py

```diff
@@ -574,33 +565,73 @@ def _initialize_roles_and_keystore(
     Returns:
         A dictionary containing entered information about taf roles and keys (total number of keys per role,
         parent roles of roles, threshold of signatures per role, indicator if metadata should be signed using
-        a yubikey for each role, key length and signing scheme for each role) and keystore file path
+        a yubikey for each role, key length and signing scheme for each role) and keystore file path.
     """
+
     roles_key_infos_dict = read_input_dict(roles_key_infos)
-    if keystore is None:
-        # if keystore path is specified in roles_key_infos and is a relative path
-        # it should be relative to the location of the file
-        # roles_key_infos can either be path to a json file, or a dictionary (or not provided)
-        if "keystore" not in roles_key_infos_dict:
-            taf_logger.info(
-                f"Keystore not specified. Using default location {default_keystore_path()}"
+
+    if not roles_key_infos_dict and enter_info:
+        roles_key_infos_dict = _enter_roles_infos(None, roles_key_infos)
+
+    # Check if all keys should be loaded from/stored to Yubikeys
+    use_yubikeys = all(
+        role_info.get("yubikey", False) for role_info in roles_key_infos_dict.values()
+    )
+
+    if not use_yubikeys:
+        if not keystore:
+            keystore = roles_key_infos_dict.get("keystore")
+            if not keystore:
+                while True:
+                    use_keystore = (
+                        input(
+                            "Do you want to save/load keys from keystore files? [y/N]: "
+                        )
+                        .strip()
+                        .lower()
+                    )
+                    if use_keystore in ["y", "n"]:
+                        break
+                if use_keystore == "y":
+                    keystore = input("Please provide keystore path: ").strip()
```

Comment:
Add `(default ./keystore)`. Let's also word this as "enter" instead of "provide"

Reply:
I made the changes and omitted the extra key generation text. 
I still get this in the output: 
...root.pub is not a file!
...root is not a file!
...targets.pub is not a file!

The command makes those files by the end of the command.